bin/station.js

```diff
@@ -21,6 +21,7 @@ Sentry.init({
 
 const core = new Core(getDefaultRootDirs())
 const modules = [
+  'runtime',
```

Comment:
Same applies to the next file, can we call it `zinnia` instead of `runtime` everywhere?

Reply:
I initially started using `zinnia` everywhere but then decided to change the module name to `runtime`.

Here is my reasoning:

- In the longer term, we want all modules to be trusted and running via Zinnia. When that happens, we will have N+2 components in Station Core: Core itself, Module runtime (Zinnia) and N modules.
- As a Station user, I don't want to learn about Station internals and that we use Zinnia as the module runtime.
- As a Station user, when inspecting logs or activity, I can imagine what "runtime" might mean. I would have no idea what "zinnia" refers to.

> What do you think about calling the module `zinnia` instead? That's what I'd intuitively query

I think you have this intuition because you are a Station developer. Do you think Station users will find this intuitive too?

Anyhow, my opinion is not strongly held. If you think we should use `zinnia` even after considering my arguments, then let's do it.

From my perspective, it's important to build a consistent user experience. If we decide to use "Zinnia", we should use it in all user-facing places - the module name in the Activity log, the module name used for Core CLI commands like `core logs`, and so on.

Reply:
Thank you for explaining your reasoning!

I think to me `runtime` is confusing since `bacalhau` also is a runtime, as `aqua` will be one. You're right that eventually we just want to have `core`, `runtime` and modules, but I see it taking a long time until the platform is both developed and rewarding enough (enough nodes) for zinnia to be the only option.

What do you think about calling it `zinnia` until it is the only runtime?

> From my perspective, it's important to build a consistent user experience. If we decide to use "Zinnia", we should use it in all user-facing places - the module name in the Activity log, the module name used for Core CLI commands like core logs, and so on.

+100, I'd call it `zinnia` _everywhere_, and nowhere `runtime`.

Wdyt?

Reply:
Fair enough, let's use `zinnia` everywhere 👌🏻